examples/bridge-app/linux/main.cpp

```diff
@@ -604,6 +552,36 @@ EmberAfStatus HandleReadPowerSourceAttribute(DevicePowerSource * dev, chip::Attr
     return EMBER_ZCL_STATUS_SUCCESS;
 }
 
+EmberAfStatus HandleReadTempMeasurementAttribute(DeviceTempSensor * dev, chip::AttributeId attributeId, uint8_t * buffer, uint16_t maxReadLength)
+{
+    if ((attributeId == ZCL_TEMP_MEASURED_VALUE_ATTRIBUTE_ID) && (maxReadLength == 2))
+    {
+        *(int16_t *) buffer = dev->GetMeasuredValue();
+    }
+    else if ((attributeId == ZCL_TEMP_MIN_MEASURED_VALUE_ATTRIBUTE_ID) && (maxReadLength == 2))
+    {
+        *(int16_t *) buffer = dev->mMin;
+    }
+    else if ((attributeId == ZCL_TEMP_MAX_MEASURED_VALUE_ATTRIBUTE_ID) && (maxReadLength == 2))
+    {
+        *(int16_t *) buffer = dev->mMax;
+    }
+    else if ((attributeId == ZCL_FEATURE_MAP_SERVER_ATTRIBUTE_ID) && (maxReadLength == 4))
+    {
+        *buffer = (uint32_t) ZCL_BRIDGED_DEVICE_BASIC_FEATURE_MAP;
```

Comment:
This doesn't make any sense: why are we casting to uint32_t, then assigning to a uint8_t?

This directory should probably enable `-Wconversion`....

Reply:
Changed to memcpy.